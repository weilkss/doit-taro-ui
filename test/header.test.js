import React from 'react'
import { renderToString } from 'nerv-server'
import { Header } from '../lib'

describe('Header', () => {
  it('render Header -- props backgroundColor(#333)', () => {
    const component = renderToString(
      <Header backgroundColor='#333'>头部</Header>
    )
    expect(component).toMatchSnapshot()
  })
})
describe('Header', () => {
  it('render Header -- props color(#fff)', () => {
    const component = renderToString(<Header color='#fff'>头部</Header>)
    expect(component).toMatchSnapshot()
  })
})
