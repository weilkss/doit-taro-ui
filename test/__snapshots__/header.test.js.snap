// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header render Header -- props backgroundColor(#333) 1`] = `undefined`;

exports[`Header render Header -- props color(#fff) 1`] = `undefined`;
